base/applications/regedit/lang/es-ES.rc

```diff
@@ -1,5 +1,5 @@
 /* Spanish Language resource file.
- * Translated by: ?? and Ismael Ferreras Morezuelas <2014-11-07> */
+ * TRANSLATOR: 2014 Ismael Ferreras Morezuelas */
```

Comment:
"??" was referred to Wine translators, as follows:

```txt
 * Copyright 2003 José Manuel Ferrer Ortiz
 * Copyright 2005 Patricio Martínez Ros
 * Copyright 2005 Samuel Serapión Vega
 *	         2008 Javier Remacha
```

From: https://github.com/reactos/reactos/blame/dfc4c1754c3224b3414395e02af9f057cb495694/reactos/base/applications/regedit/lang/es-ES.rc